### PR TITLE
Reverting rating code

### PR DESCRIPTION
**Pull request by dallen68**

### CODE REVIEW

File: src/main/java/com/google/step/servlets/PromoCodeServlet.java [hunk 1]
```diff
@@ -1,5 +1,18 @@
...
 
-public class PromoCodeServlet {
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
 
+@WebServlet("/promo-code")
+public class PromoCodeServlet extends HttpServlet {
+
```
Comment (josefj1519) [inline src/main/java/com/google/step/servlets/PromoCodeServlet.java]: I believe we need an init function

Comment (dallen68) [reply src/main/java/com/google/step/servlets/PromoCodeServlet.java]: I don't think that's the case with servlets.

File: src/main/java/com/google/step/servlets/PromoCodeServlet.java [hunk 2]
```diff
@@ -1,5 +1,18 @@
...
+    @Override
+    public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    }
+
+    @Override
+    public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    }
```
Comment (margaretchan) [inline src/main/java/com/google/step/servlets/PromoCodeServlet.java:17]: I don't believe we need a doPost in the MVP

File: src/main/java/com/google/step/servlets/PromoCodeServlet.java (hunk 1 above)
Comment (josefj1519) [reply src/main/java/com/google/step/servlets/PromoCodeServlet.java]: It's not needed for servlets, but we want it to initialize the youtubescraper.

File: src/main/webapp/script.js [hunk 3]
```diff
@@ -1,33 +1,26 @@
...
-    fetch('/promo-code?rating=' + rating + '&videoUrl=' + videoUrl, {method: 'POST'});
-    displayCodes();
-}
+//This function is not currently in use but may be used in further implementations
+// function ratePromoCode(tableRow, rating) {
+//     var videoUrl = document.getElementById('promoCodeTable').rows[tableRow].cells[0].firstChild.getAttribute("href");
+//     console.log(videoUrl);
+//     fetch('/promo-code?rating=' + rating + '&videoUrl=' + videoUrl, {method: 'POST'});
+//     getCodes();
+// }
```
Comment (margaretchan) [inline src/main/webapp/script.js]: Delete this function, version control will remember it if you need it for future implementation. Also explained in Clean Code chapter 4 section "Commented-Out Code" (pg. 68)

**Review by margaretchan — CHANGES_REQUESTED** [review]

**Review by josefj1519 — APPROVED** [review]

**Review by margaretchan — APPROVED** [review]

**Pull request merged**